installer/py3.10-darwin-arm64-mps-reqs.txt

```diff
@@ -1816,7 +1822,7 @@ toolz==0.12.0 \
     --hash=sha256:2059bd4148deb1884bb0eb770a3cde70e7f954cfbbdc2285f1f2de01fd21eb6f \
     --hash=sha256:88c570861c440ee3f2f6037c4654613228ff40c93a6c25e0eba70d17282c6194
     # via altair
-torch==1.13.0 \
+torch==1.13.0 ; platform_system == "Darwin" \
```

Comment:
why is torch 1.13 being pinned on macos?

Reply:
It's actually what we released, although I really don't understand how... It makes me think that the performance regression is in torchvision instead

Reply:
it's not, it's in torch. here is the `pip list` output from the conda env with good performance:
```
torch                         1.12.1
...
torchvision                   0.13.1
```